comments.go

```diff
@@ -47,6 +47,32 @@ func (c *CommentsController) Show(ctx *app.ShowCommentsContext) error {
 	})
 }
 
+// Update does PATCH comment
+func (c *CommentsController) Update(ctx *app.UpdateCommentsContext) error {
+	id, err := uuid.FromString(ctx.ID)
+	if err != nil {
+		jerrors, _ := jsonapi.ErrorToJSONAPIErrors(goa.ErrUnauthorized(err.Error()))
+		return ctx.BadRequest(jerrors)
+	}
+
+	return application.Transactional(c.db, func(appl application.Application) error {
+		cm, err := appl.Comments().Load(ctx.Context, id)
+		if err != nil {
+			return jsonapi.JSONErrorResponse(ctx, err)
+		}
+		cm.Body = *ctx.Payload.Data.Attributes.Body
```

Comment:
We can use the this function to get current user ID: `login.ContextIdentity(ctx)` and this value can be compared against `created_by` value.